nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    'Not Implemented Error'
+  end
```

Comment:
- [ ] Good job adding the error message. I think you can make it more comprehensive and specific by adding something like this 👇🏽 

```
   def correct_name
        raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
    end
```

Reply:
i think this could be optional ?
@vickymarz